src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.Dev.csproj

```diff
@@ -0,0 +1,685 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This project seems largely duplicative of `Xamarin.Android.Build.Tasks.csproj`. Is there some way we could instead `<Import/>` `Xamarin.Android.Build.Tasks.csproj` and override some properties? Or perhaps we should turn `Xamarin.Android.Build.Tasks.csproj` into a `.shproj`?

Reply:
I did try, but shared projects do not seem to like the way we include stuff like the Linker code and the pdb stuff etc. It seems they don't like the items where we define a property for the path e.g.`$(MonoSourceFullPath)`.  I will have another go.. 

It might be worth splitting those items into a separate .targets file, since we never edit them anyway